action.yml

```diff
@@ -6,120 +6,44 @@ branding:
   icon: 'package'
 
 inputs:
+  project:
+    description: 'The name or ID of the project associated with this release.'
+    required: true
   api_key:
-    description: 'The API key used to access Octopus Deploy. You must provide an API key or username and password. If the guest account is enabled, a key of API-GUEST may be used. It is strongly recommended that this value retrieved from a GitHub secret.'
-  cancel_on_timeout:
-    default: false
-    description: 'Cancel the deployment if the deployment_timeout is exceeded (the default timeout is 10 minutes).'
+    description: 'The API key used to access Octopus Deploy. An API key is required, but you may also use the OCTOPUS_API_KEY environment variable. If the guest account is enabled, a key of API-GUEST may be used. It is strongly recommended that this value retrieved from a GitHub secret.'
```

Comment:
Maybe we should be more stern that we strongly recommend using ENV over the tag. Any suggestions on how we should word this? @jbristowe

Reply:
As per call, I've updated the example to prefer env, matching push-package-action, and also added environment vars in a specific table to match that too